packages/periphery/test/int/eth/MarginlyCompositeOracle.spec.ts

```
@@ -0,0 +1,68 @@
+import { ethers } from 'hardhat';
+import { MarginlyCompositeOracle, PendleMarketOracle, PythOracle } from '../../../typechain-types/contracts/oracles';
+import { getDecimalsDiff, printPrices } from '../../shared/common';
+
+describe.only('Composite oracle PT-wstUSR/USR with Pendle for PT-wstUSR/wstUSR, Pyth wstUSR/USR', () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)